### PR TITLE
Observed addresses added

### PR DESCRIPTION
**Pull request by KonstantinRyazantsev**

### CODE REVIEW

File: src/Indexer.Common/Indexer.Common.csproj [hunk 1]
```diff
@@ -64,11 +66,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.4" />
-
-    <PackageReference Include="Swisschain.Extensions.Idempotency.EfCore" Version="1.0.6" />
-    <PackageReference Include="Swisschain.Extensions.Idempotency.MassTransit" Version="1.0.3" />
-    <PackageReference Include="Swisschain.Sirius.Sdk.Crypto" Version="2.2.3" />
-    <PackageReference Include="Swisschain.Sirius.Sdk.Integrations.Client" Version="1.0.0-beta.17" />
+    <PackageReference Include="Swisschain.Sirius.Sdk.Crypto" Version="2.2.5" />
```
Comment (OttoVT) [inline src/Indexer.Common/Indexer.Common.csproj:69]: use the latest version here =)

File: src/Indexer.Common/Persistence/EntityFramework/CommonDatabaseContext.cs [hunk 2]
```diff
@@ -53,7 +53,7 @@ private void BuildAssets(ModelBuilder modelBuilder)
                 e.HasKey(x => x.Id);
 
                 e.Property(x => x.Id)
-                    .HasIdentityOptions(100_000)
+                    .HasIdentityOptions(300_000_000)
```
Comment (OttoVT) [inline src/Indexer.Common/Persistence/EntityFramework/CommonDatabaseContext.cs:56]: use 100_000_000 instead

**Review by OttoVT — CHANGES_REQUESTED** [review]

File: src/Indexer.Common/Persistence/EntityFramework/CommonDatabaseContext.cs (hunk 2 above)
Comment (KonstantinRyazantsev) [reply src/Indexer.Common/Persistence/EntityFramework/CommonDatabaseContext.cs:56]: No, we'll use diferent prefixes for each service. 1 for brokerage, 2 for executor, 3 for indexer

**Review by OttoVT — APPROVED** [review]

File: src/Indexer.Common/Indexer.Common.csproj (hunk 1 above)
Comment (KonstantinRyazantsev) [reply src/Indexer.Common/Indexer.Common.csproj:69]: done in the #32

**Pull request merged**